examples/models/llama2/export_llama_lib.py

```diff
@@ -245,7 +245,7 @@ def quantize(
 
     if qmode == "int8":
         # Add quantization mode options here: group size, bit width, etc.
-        return WeightOnlyInt8QuantHandler(model).quantized_model()
+        return WeightOnlyInt8QuantHandler(model).quantize()
```

Comment:
since you are changing this, could you change it to `WeightOnlyInt8QuantHandler().quantize(model)`? that will align better with our plan for the unified quantization API.

Reply:
This is about as far as I think we can go, without risk.  (renaming a method is one thing. switching the whole flow is too many moving parts.... maybe we just leave that for after the release?)